### DISCUSSION TITLE
[@types/three] examples library stats.end should return a number

### DISCUSSION THREAD

**Discussion by IARI** [DefinitelyTyped/DefinitelyTyped#61861] [Issues with a @types package] [answered]
The interface `Stats` declares `end(): void;` here:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/three/examples/jsm/libs/stats.module.d.ts

However, the implementation does return a number (timestamp)
https://github.com/mrdoob/stats.js/blob/master/src/Stats.js#L93

Hence the interface `Stats` should declare `end(): number;`

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
I just opened #61869
